.github/CONTRIBUTNG.md

```diff
@@ -0,0 +1,31 @@
+# Contributing
+
+Contributions are welcome, and they are greatly appreciated! Every little bit helps, and credit will always be given.
+
+Please read the PAVICS [Developer Documentation](https://pavics-sdi.readthedocs.io/en/latest/dev/index.html) to get started.
+
+## Policy
+Since PAVICS is used in production by multiple organizations, this deployment repository also has a policy regarding contributions. 
+
+### Policy objectives:
+1. Keep production stable between upgrades
+2. Encourage PR to be merged more quickly
+3. Achieve the two previous objectives 
+      * by weighing down everyone's workflow as little as possible
+      * by having an approach that scales well as more nodes go into production and the number of PR increases
+
+### Policy rules:
+1. The repository has a main branch, "master", open to the community where contributions, "PR", are welcome. This master branch must not have owners and therefore no organization can block contributions to it.
+2. Contributions should be backward-compatible whenever possible, or feature a toggle switch so that organizations can activate the new feature on their own schedule.
+3. Contributions will trigger a test suite that must successfully pass before being reviewed.
+4. Contributions must be reviewed by at least one organization and possibly more than one organization depending on the impact of the contribution.
+5. The reviews must be rigorous while respecting the initial scope.
+6. Each organization wishing to review has the duty to do so within a reasonable period of time (7 days) or to indicate its intention to do so later with reasonable reasons (e.g., vacation). After this time, its implicit support will be considered. It will be assumed that the organization agrees to the changes and there will be a merge without further notice.
```

Comment:
"to review" -> to review the changes
"and there will be a merge" -> and they will get merged